src/components/MultiHopTrade/components/LimitOrderV2/InnerSteps.tsx

```diff
@@ -161,6 +162,7 @@ export const InnerSteps = ({ isLoading }: InnerStepsProps) => {
             accountId={sellAccountId}
             stepSource={undefined} // no swapper base URL here, this is an allowance Tx
             quoteSwapperName={SwapperName.CowSwap}
+            txStatus={TxStatus.Unknown}
```

Comment:
This feels off - even though we don't need it, we should pass the actual status for the sake of correctness, if we're going to go with a `txStatus` prop.

Reply:
Honestly I removed this one, it's probably a non issue anyway and was already supported by stepSource being undefined